File: coco/src/main/java/coco/ep/m6/finalized/Neg.java
package coco.ep.m6.finalized;

public class Neg extends Exp implements coco.ep.m6.Neg<Exp> {

    protected Exp inner;

    public Neg(Exp _inner) {
        this.inner = _inner;
    }

    public Exp getInner() {
        return inner;
    }
    
    public void setInner(coco.ep.Exp<Exp> _inner) {
        inner = convert(_inner);
    }
}

File: coco_csharp/CoCo/EP/M4/Finalized/Neg.cs
namespace EP.M4.Finalized {
    public class Neg : Exp, EP.M4.Neg<Exp> {
        private EP.Exp<Exp> inner;

        public Neg(EP.Exp<Exp> _inner) {
            this.inner = _inner;
        }

        public EP.Exp<Exp> getInner() { return this.inner; }
        public void setInner(EP.Exp<Exp> _inner) { this.inner = _inner; }
    }
}